server/tests/oauth.specs/social.auth.spec.js

```diff
@@ -0,0 +1,80 @@
+import chai, { expect } from "chai";
+import chaiHttp from "chai-http";
+import app from "../../index";
+import database from "../../models/index";
+
+const User = database["Users"];
+
+const baseUrl = "/api/v1";
+chai.use(chaiHttp);
+
+describe("Social Platform Authentication Test", () => {
+  describe("Test for Endpoints", () => {
```

Comment:
This describe block will only make sense if you have multiple tests. Since you have only one test under `it`, please just use the it block